
 This patch fixes the despooling size printed in the Job 
 report. Previously, it reported the total size for the
 device. This patch makes it report the size for the job.
 Apply the patch to 1.36.1 with:

 cd <bacula-source>
 patch -p0 <1.36.1-spool.patch
 make
 make install
 ...

Index: src/stored/spool.c
===================================================================
RCS file: /cvsroot/bacula/bacula/src/stored/spool.c,v
retrieving revision 1.22
diff -u -r1.22 spool.c
--- src/stored/spool.c	16 Oct 2004 11:51:32 -0000	1.22
+++ src/stored/spool.c	18 Dec 2004 20:39:02 -0000
@@ -201,7 +201,7 @@
    Dmsg0(100, "Despooling data\n");
    Jmsg(jcr, M_INFO, 0, _("%s spooled data to Volume. Despooling %s bytes ...\n"),
         commit?"Committing":"Writing",
-	edit_uint64_with_commas(jcr->dcr->dev->spool_size, ec1));
+	edit_uint64_with_commas(jcr->dcr->spool_size, ec1));
    dcr->spooling = false;
    lock_device(dcr->dev);
    dcr->dev_locked = true; 
